### ISSUE TITLE
Shift knob does not respect double-click reset

### ISSUE THREAD

**Issue by zezic** [zezic/ZZC-Phaseque#16]

**Comment by VegaDeftwing** [comment]
This is true for all knobs on the right side, V, Shift, Len, In, Out. It's also true of the X/Y for the Expr box.

**Comment by VegaDeftwing** [comment]
I now have code locally which fixes all knob resets as well as the X-Y for expression reset, but I have it based on #24 so I'll wait to commit it.

**Comment by VegaDeftwing** [comment]
b069fc4a416ffd5ea3f96b6109f0a0c027764759
